docs/Chipyard-Basics/Initial-Repo-Setup.rst

```diff
@@ -6,28 +6,53 @@ Requirements
 
 Chipyard is developed and tested on Linux-based systems.
 
-.. Warning:: It is possible to use this on macOS or other BSD-based systems, although GNU tools will need to be installed; it is also recommended to install the RISC-V toolchain from ``brew``.
+.. Warning:: It is possible to use this on macOS or other BSD-based systems, although GNU tools will need to be installed;
+    it is also recommended to install the RISC-V toolchain from ``brew``.
 
 .. Warning:: Working under Windows is not recommended.
 
+Running on AWS EC2 with FireSim
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-In CentOS-based platforms, we recommend installing the following dependencies:
+If you plan on using Chipyard alongside FireSim, you should refer to the :fsim_doc:`FireSim documentation <>`.
+Specifically, you should follow the :fsim_doc:`Initial Setup/Simulation <Initial-Setup/index.html>`
+section of the docs up until :fsim_doc:`Setting up the FireSim Repo <Initial-Setup/Setting-up-your-Manager-Instance.html#setting-up-the-firesim-repo>`.
+At that point, instead of checking out FireSim you can checkout Chipyard by following :ref:`Chipyard-Basics/Initial-Repo-Setup:Setting up the Chipyard Repo`.
```

Comment:
```suggestion
At that point, instead of cloning FireSim you can clone Chipyard by following :ref:`Chipyard-Basics/Initial-Repo-Setup:Setting up the Chipyard Repo`.
```